### PR TITLE
Add license badge and sbt requirements

### PR DESCRIPTION
**Pull request by bdarfler**

### CODE REVIEW

**Comment by bdarfler** [comment]
@ahepburn added a license badge and a requirements section

File: README.md [hunk 1]
```diff
@@ -5,7 +5,13 @@ Support for running [DynamoDB Local](http://docs.aws.amazon.com/amazondynamodb/l
...
 Based on the [Maven Plugin for DynamoDB](https://github.com/jcabi/jcabi-dynamodb-maven-plugin).
 
-Initial [implementation](https://github.com/grahamar/sbt-dynamodb) by [Graham Rhodes](https://github.com/grahamar).  
+Initial [implementation](https://github.com/grahamar/sbt-dynamodb) by [Graham Rhodes](https://github.com/grahamar). 
+
+[![MIT license](https://img.shields.io/badge/license-MIT%20License-blue.svg)](LICENSE) 
+
+Requirements
+------------
+sbt 0.13.5+
```
Comment (ahepburn) [inline README.md]: I think you mentioned this was 0.13.6+? (or perhaps just list the resolver to include if using a prior version)

Comment (bdarfler) [inline README.md]: Yah, I'll bump this to 0.13.6+. There is probably some crazy way to get the bintray resolver before then but the plugin that does that targets 0.13.6+ as well (chicken and egg)

**Comment by ahepburn** [comment]
:shipit: looks good

**Pull request merged**